src/mesh/azext_mesh/servicefabricmesh/mgmt/servicefabricmesh/version.py

```diff
@@ -1,12 +0,0 @@
-# coding=utf-8
```

Comment:
version.py is used here: https://github.com/Azure/azure-cli-extensions/blob/b5e01e2262234bb60152e4d852b357ee849558dd/src/mesh/azext_mesh/servicefabricmesh/mgmt/servicefabricmesh/__init__.py#L13